package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@equinor/fusion-components",
-    "version": "2.12.2",
+    "version": "2.13.0",
```

Comment:
Please us a patch version bump for this release

Reply:
fixed in [19e4d53](https://github.com/equinor/fusion-components/pull/1691/commits/19e4d53af5ba0c006bf55cb90c888bf6d27fdb69)